tests/core/dom/documentfragment.js

```diff
@@ -209,6 +209,40 @@
 			assert.isTrue( inner2.equals( childNodesList.getItem( 1 ) ) );
 		},
 
+		// (#2021)
+		test_find: function() {
+			var frag = new CKEDITOR.dom.documentFragment( CKEDITOR.document ),
+				b1 = new CKEDITOR.dom.element( 'b' ),
+				b2 = new CKEDITOR.dom.element( 'b' ),
+				em = new CKEDITOR.dom.element( 'em' ),
+				found;
+
+			frag.append( b1 );
+			frag.append( b2 );
+			frag.append( em );
+
+			found = frag.find( 'b' );
+			assert.areEqual( 2, found.count() );
+			assert.isTrue( b1.equals( found.getItem( 0 ) ) );
+			assert.isTrue( b2.equals( found.getItem( 1 ) ) );
```

Comment:
There are no fail messages for 2 identically reported (if failed) assertions.

Reply:
Remember that we have this customization of `assert.areEqual` that works with our DOM wrappers too.

```js
assert.areEqual( b2, found.getItem( 1 ), 'Item #1' );
```